src/progress_bar.rs

```diff
@@ -558,6 +558,14 @@ impl Default for MultiProgress {
     }
 }
 
+enum InsertLocation<'a> {
```

Comment:
Style nit: please move this definition down below the `impl WeakProgressBar`.

Reply:
Done